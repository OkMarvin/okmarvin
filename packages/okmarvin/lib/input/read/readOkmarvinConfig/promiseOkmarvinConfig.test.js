const promiseOkmarvinConfig = require('./promiseOkmarvinConfig')
const path = require('path')
test('returns {} when configuration file does not exist', () => {
  promiseOkmarvinConfig(path.join(__dirname, '..', 'fixtures', 'not-exist')).then(
    data => {
      expect(data).toEqual({})
    }
  )
})
test('returns module content when configuration file do exist', () => {
  promiseOkmarvinConfig(path.join(__dirname, '..', 'fixtures')).then(data => {
    expect(data).toEqual({
      markdown: {
        toc: {
          title: '目录'
        },
        loadLanguages: ['bash', 'elixir']
      }
    })
    expect(data).toMatchSnapshot()
  })
})
