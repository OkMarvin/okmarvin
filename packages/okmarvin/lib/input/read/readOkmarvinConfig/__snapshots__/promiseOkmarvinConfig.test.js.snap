// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns module content when configuration file do exist 1`] = `
Object {
  "markdown": Object {
    "loadLanguages": Array [
      "bash",
      "elixir",
    ],
    "toc": Object {
      "title": "目录",
    },
  },
}
`;
